tests/end_to_end/L1L2Messaging/test_messaging.py

```diff
@@ -125,9 +164,8 @@ async def test_should_fail_unauthorized_message_sender(
         msg_counter_before = await message_app_l2.receivedMessagesCounter()
         await invoke(
             "kakarot",
-            "set_authorized_message_sender",
-            int(l1_kakarot_messaging.address, 16),
-            False,
+            "set_l1_messaging_contract_address",
+            0,
```

Comment:
```suggestion
            0
```

and I think that linter will put all in the same line